components/identities/updateProfilePic.tsx

```diff
@@ -50,8 +50,22 @@ const UpdateProfilePic: FunctionComponent<UpdateProfilePicProps> = ({
             selectedPfp={selectedPfp}
             userNfts={userNfts}
             isLoading={isLoading}
-            title="Our Suggestions"
+            title="Choose your NFT Profile picture"
           />
+          <div className={`hidden justify-center mt-4 ${userNfts && "flex"}`}>
```

Comment:
It's a bad practice to use css to render dynamically in react. 
Instead use React conditional rendering.
e.g.:
```tsx
{
userNfts ?
 <div>
... 
</div>
: null
}